test/acceptance/behave/components/ngsiv2/entities/list_all_entities.feature

```diff
@@ -0,0 +1,533 @@
+      | 20     |
+
+  @only_offset_error
+  Scenario Outline:  list all entities in NGSI v2 with only wrong offset query parameter
+    Given  a definition of headers
+      | parameter          | value                       |
+      | Fiware-Service     | test_list_only_offset_error |
+      | Fiware-ServicePath | /test                       |
+      | Content-Type       | application/json            |
+    And create "10" entities with "2" attributes
+      | parameter        | value                   |
+      | entities_type    | room                    |
+      | entities_id      | room2                   |
+      | attributes_name  | timestamp               |
+      | attributes_value | 017-06-17T07:21:24.238Z |
+      | attributes_type  | date                    |
+      | metadatas_number | 2                       |
+      | metadatas_name   | very_hot                |
+      | metadatas_type   | alarm                   |
+      | metadatas_value  | random=10               |
+    And verify that receive several "Created" http code
+    When get all entities
+      | parameter | value    |
+      | offset    | <offset> |
+    Then verify that receive an "Bad Request" http code
+    And verify an error response
+      | parameter   | value                                                        |
+      | error       | BadRequest                                                   |
+      | description | Bad pagination offset: /<offset>/ [must be a decimal number] |
+    Examples:
+      | offset |
+      | -3     |
+      | 1.5    |
+      | ewrw   |
+      | <2     |
+      | %@#    |
+
+  @limit_offset
+  Scenario Outline:  list all entities in NGSI v2 with limit and offset queries parameters
+    Given  a definition of headers
+      | parameter          | value                  |
+      | Fiware-Service     | test_list_limit_offset |
+      | Fiware-ServicePath | /test                  |
+      | Content-Type       | application/json       |
+    And create "10" entities with "2" attributes
+      | parameter        | value                   |
+      | entities_type    | room                    |
+      | entities_id      | room2                   |
+      | attributes_name  | timestamp               |
+      | attributes_value | 017-06-17T07:21:24.238Z |
+      | attributes_type  | date                    |
+      | metadatas_number | 2                       |
+      | metadatas_name   | very_hot                |
+      | metadatas_type   | alarm                   |
+      | metadatas_value  | random=10               |
+    And verify that receive several "Created" http code
+    When get all entities
+      | parameter | value    |
+      | limit     | <limit>  |
+      | offset    | <offset> |
+    Then verify that receive an "OK" http code
+    And verify that all entities are returned
+    Examples:
+      | limit | offset |
+      | 1     | 1      |
+      | 20    | 1      |
+      | 1     | 20     |
+      | 5     | 3      |
+
```

Comment:
I think the feature is quite complete. However, I think it could be enhanced taking into account cases with compount attributes values (JSON vectors and object) and metadata compound values (JSON vector and objects, these ones in "skipped" status yet, as the functionality is not get currently implemented).

I think we have the "Examples" for similar cases in other .feature file, so it could be easy.